src/manifest_extra.firefox.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Firefox actually [doesn't require ID since version 48](https://extensionworkshop.com/documentation/develop/extensions-and-the-add-on-id/#when_do_you_need_an_add-on_id). Since the current ID is not using 'Openverse' branding, we could consider dropping it altogether.

Reply:
Thanks for pointing this. Love any excuse to delete code.